unreal_asset/src/properties/mod.rs

```diff
@@ -732,7 +732,10 @@ impl Property {
                     .into()
             }
 
-            "MulticastDelegateProperty" => MulticastDelegateProperty::new(
+            "MulticastDelegateProperty"
+            | "DelegateProperty"
+            | "MulticastSparseDelegateProperty"
+            | "MulticastInlineDelegateProperty" => MulticastDelegateProperty::new(
```

Comment:
this will break serializing back because every delegate property will nwo be serialized as a multicast one

Reply:
But aren't they serialised the same way?

Reply:
serialization is the same except for the name that is written, and I really don't want to break compatibility just in case

Reply:
oh wait yeah they are typed in writing

Reply:
okay I'll have to make new types for them then and then I'll do a new PR